### PR TITLE
PoC realyer native-bitcoin

### PR DESCRIPTION
**Pull request by sczembor**
## Description

PoC native-bitcoin . Simple script that takes hex encoded bitcoin transaction and broadcasts it to the network

closes: #17 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary

### CODE REVIEW

File: cmd/native-btc/main.go [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+	if err != nil {
+		fmt.Println("Error loading .env file", err)
+		return
+	}
+
+	if len(os.Args) < 2 {
+		fmt.Println("Missing transaction file path")
+		return
+	}
+	txFilePath := os.Args[1]
```
Comment (udit-gulati) [inline cmd/native-btc/main.go:32]: Please add a sample transaction file.

File: cmd/native-btc/main.go [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+	"github.com/btcsuite/btcd/rpcclient"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/joho/godotenv"
+)
+
+// ENV variables
+const (
+	Host = "BTC_RPC"
+	User = "USER_RPC"
+	Pass = "PASS_RPC"
```
Comment (udit-gulati) [inline cmd/native-btc/main.go:18]:
```suggestion
	BtcRpcHost = "BTC_RPC"
	BtcRpcUser = "USER_RPC"
	BtcRpcPass = "PASS_RPC"
```

File: cmd/native-btc/main.go [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+)
+
+func main() {
+	err := godotenv.Load("../../.env")
```
Comment (udit-gulati) [inline cmd/native-btc/main.go:22]: Do a check here if the file exists, if not then error that `.env` file missing on path `/path/to/env/file`

File: README.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ A software that monitors and relayers:
 - CometBFT blocks to update Native -> Pera light client
 - Bitcoin blocks to update Bitcoin -> Native light client
 - Bitcoin SPV proofs to verify dWallet holdings
+- Native -> Bitcoin transaction relayer
```
Comment (udit-gulati) [inline README.md:22]: Please add the steps in README on how to run the script.

1. Copy `.env.example` file to `.env`
2. Update values in `.env` file
3. Run script with path of transaction file as command line argument

**Review by udit-gulati — CHANGES_REQUESTED** [review]

File: cmd/native-btc/main.go (hunk 1 above)
Comment (sczembor) [reply cmd/native-btc/main.go:35]: As disscussed earlier

File: cmd/native-btc/main.go (hunk 3 above)
Comment (sczembor) [reply cmd/native-btc/main.go:22]: Done

File: README.md (hunk 4 above)
Comment (sczembor) [reply README.md:22]: Done

**Review by udit-gulati — APPROVED** [review]

**Pull request merged**